rts/lra/lra-coordinator-jar/src/main/java/io/narayana/lra/coordinator/domain/service/LRAService.java

```diff
@@ -67,7 +67,23 @@
 
     public Transaction getTransaction(URI lraId) throws NotFoundException {
         if (!lras.containsKey(lraId)) {
+            String uid = getUid(lraId);
+
+            // try comparing on uid since differnt URIs can map to the same resource
+            // (eg localhost versus 127.0.0.1 versus :1 etc)
+            for (Transaction lra : lras.values()) {
```

Comment:
I think this is perhaps best recorded as a different JBTM - but thanks for the comment as it explains the motiviation.

Reply:
Okay I added a new JIRA https://issues.jboss.org/browse/JBTM-3222 and updated the PR description

Reply:
Thanks Mike, I think it needs to be in a separate commit

Reply:
I pulled out the changes and moved them into a separate commit.